Dockerfile

```diff
@@ -0,0 +1,18 @@
+FROM debian:stretch
+
+COPY . /usr/src/kafkacat
+
+RUN runDeps=''; \
```

Comment:
I'll drop the runDeps. This whole block is just copy paste from a convention I tend to use on debian.